tests/components/recorder/test_init.py

```diff
@@ -55,7 +57,7 @@ def test_saving_state(self):
             assert db_states[0].event_id > 0
             state = db_states[0].to_native()
 
-        assert state == self.hass.states.get(entity_id)
+        assert state == _state_empty_context(self.hass, entity_id)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)